Source/Core/InputCommon/InputCommon.vcxproj.filters

```diff
@@ -202,8 +203,9 @@
     <ClInclude Include="ControlReference\ControlReference.h">
       <Filter>ControllerInterface</Filter>
     </ClInclude>
+    <ClInclude Include="InputProfile.h" />
   </ItemGroup>
   <ItemGroup>
     <Text Include="CMakeLists.txt" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
Why remove the newline?

Reply:
Hmm, I don't believe it was intentional, I can revert this change.